locale/ru-ru/texts.po

```diff
@@ -492,6 +492,22 @@ msgstr "Доступный Баланс"
 msgid "Locked"
 msgstr "Заблокированный"
 
+#.  translator: Used when the QR Code Scanner is opened, and user will manually
+#.  enter the information.
+#: src/screens/NanoContractRegisterQrCodeScreen.js:26
+#: src/screens/RegisterToken.js:27 src/screens/SendScanQRCode.js:87
+msgid "Manual info"
+msgstr "Ручной ввод"
```

Comment:
Was this an automatic translation? Maybe remove?

Reply:
No, this was already there. I believe it was only moved.